packages/headless/src/integration-tests/commerce-placements.test.ts

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable @cspell/spellchecker */
+import {buildCommercePlacementsEngine} from '../app/commerce-placements-engine/commerce-placements-engine';
+import {
+  getBadge,
+  getRecs,
+} from '../features/placement-set/placement-set-action';
+import {getOrganizationEndpoints} from '../recommendation.index';
+
+describe('commerce placements', () => {
+  it('should allow to query without errors', async () => {
+    const engine = buildCommercePlacementsEngine({
+      configuration: {
+        // random min privileges key in dev
+        accessToken: 'xxa0ccd787-ba54-40fb-89ff-f393356448c4',
+        // random test org in dev
+        organizationId: 'aduiorgtestdonotdeletepleaseas62tcf4',
+        organizationEndpoints: getOrganizationEndpoints(
+          'aduiorgtestdonotdeletepleaseas62tcf4',
+          'dev'
+        ),
+      },
+    });
+
+    await engine.dispatch(
+      getBadge({
+        placementId: '5GKB9hFRTnmC24QxZTlDoA',
+        implementationId: 'CB8mj48FSaC8iWMFTZYJgg',
+        seeds: [{ids: ['SP00553_00002'], src: 'pdp'}],
+      })
+    );
+
+    await engine.dispatch(
+      getRecs({
+        previewOpts: {
+          campaignId: 'CHLR7FPrRTi5bq_oQcCUcA',
+        },
+        placementId: 'kTLIHvXYTyWvuej4qJJaiA',
+        implementationId: 's1rAcVo_SmCVoSpEZ',
```

Comment:
How did you get these values for `campaignId`, `placementId` and `implementationId`? Is it easy for an implementer to get them?

Reply:
At the moment, it's trial and error, and no, it's not easy to get them or to know what they are for. Baby steps :D

First goal is to go without any magic, and see where we can simplify/how we could make this easier.

Probably some of these should be global engine config, not on a per request kinda thing.